packages/neuron-ui/src/components/NetworkEditor/hooks.ts

```diff
@@ -126,72 +120,54 @@ export const useHandleSubmit = (
 ) =>
   useCallback(async () => {
     const warning = {
-      type: 'warning',
+      type: 'warning' as 'warning',
       timestamp: Date.now(),
       content: '',
     }
     if (!name) {
-      return dispatch({
-        type: AppActions.AddNotification,
-        payload: {
-          ...warning,
-          content: i18n.t(Message.NameRequired),
-        },
-      })
+      return addNotification({
+        ...warning,
+        content: i18n.t(Message.NameRequired),
+      })(dispatch)
     }
     if (name.length > MAX_NETWORK_NAME_LENGTH) {
-      return dispatch({
-        type: AppActions.AddNotification,
-        payload: {
-          ...warning,
-          content: i18n.t(Message.LengthOfNameShouldBeLessThanOrEqualTo, {
-            length: MAX_NETWORK_NAME_LENGTH,
-          }),
-        },
-      })
+      return addNotification({
```

Comment:
When validating a form, if an filed has error, do we show any error style for that field? For example, if name's length exceeds limit, beside the error message (notification), do we style the name field so it stands out (e.g., its border is rendering as red)?

If not, and if some form errors hide in the pane, the user would be confused why we don't let them submit.

Reply:
The error messages will be appended to the field in the future